File: Contracts/DataContracts/Extension.cs
using System;
using System.Collections.Generic;

namespace CentralAdministration.Contracts.DataContracts
{
    public class Extension
    {
        public Guid Id { get; set; }
        public License License { get; set; }
        public Package Package { get; set; }

    }

    public class Package
    {
        public string Version { get; set; }
        public DateTime? LastUpdatedDate { get; set; }
        public DateTime? InstalledDate { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public Guid UpdatedBy { get; set; }
    }
}

File: Module/Contracts/Extension.ts
namespace CentralAdministration.Contracts.DataContracts {
    export class Extension {
        public id: string;
        public license: License;
        public package: Package;
    }

    export class Package {
        public version: string;
        public lastUpdatedDate?: Date | null;
        public installedDate?: Date | null;
        public name: string;
        public description: string;
        public updatedBy: string;
    }
}
